common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/utils/checkins/FakeCheckinIntervalSpecification.java

```diff
@@ -0,0 +1,42 @@
+package app.coronawarn.server.common.persistence.service.utils.checkins;
+
+import app.coronawarn.server.common.protocols.internal.pt.CheckIn;
+import java.security.SecureRandom;
+import java.util.Random;
+import java.util.function.Function;
+
+public class FakeCheckinIntervalSpecification {
+
+  private static final Random RANDOM = new SecureRandom();
+
+  /**
+   * https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp
+   * /docs/spec/event-registration-server.md#generating-fake-tracetimeintervalwarnings
+   */
+  public static final Function<CheckIn, Integer> START_INTERVAL_GENERATION = (checkin) -> {
+    int period = checkin.getEndIntervalNumber() - checkin.getStartIntervalNumber();
```

Comment:
calculating start interval differs from what is mentioned in the spec. I think just the naming got accidently switched

Reply:
Yes, a mistake there. The namings of the two should be swapped.